simapp/encoding.go

```diff
@@ -1,13 +1,13 @@
 package simapp
 
 import (
-	"github.com/cosmos/cosmos-sdk/simapp/params"
+	simapparams "github.com/cosmos/cosmos-sdk/simapp/params"
 	"github.com/cosmos/cosmos-sdk/std"
 )
 
 // MakeEncodingConfig creates an EncodingConfig for testing
-func MakeEncodingConfig() params.EncodingConfig {
-	encodingConfig := params.MakeEncodingConfig()
+func MakeEncodingConfig() simapparams.EncodingConfig {
```

Comment:
ditto s/simapparams/simappparams

Reply:
yes, it was already there. (import didn't change here). I will update to tripple p